### PR TITLE
Nc Fix: locked view ui changes

### PR DESCRIPTION
**Pull request by rameshmane7218**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

File: packages/nc-gui/components/general/LockedViewFooter.vue [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  const { close } = useDialog(resolveComponent('DlgLockView'), {
+    'modelValue': isOpen,
+    'onUpdate:modelValue': closeDialog,
+  })
+
+  function closeDialog() {
+    isOpen.value = false
+    close(1000)
+  }
+}
```
[the remark this answers is not in the document]
Comment (rameshmane7218) [reply packages/nc-gui/components/general/LockedViewFooter.vue:17]: Yes thats what we want

File: packages/nc-gui/components/smartsheet/toolbar/ColumnFilter.vue [hunk 2]
```diff
@@ -1023,6 +1049,15 @@ const changeToDynamic = async (filter, i) => {
...
     <slot />
+
+    <GeneralLockedViewFooter
+      v-if="isLockedView && !nested"
+      class="-mb-2 -ml-4"
+      :class="{
+        'mt-2': !visibleFilters || !visibleFilters.length,
+      }"
+      @on-open="isOpen = false"
+    />
```
[the remark this answers is not in the document]
Comment (rameshmane7218) [reply packages/nc-gui/components/smartsheet/toolbar/ColumnFilter.vue:1060]: Only in script we have to use refVar.value not in template

File: packages/nc-gui/components/smartsheet/toolbar/ColumnFilterMenu.vue [hunk 3]
```diff
@@ -67,21 +65,15 @@ eventBus.on(async (event, column: ColumnType) => {
...
       </template>
 
-      <NcButton
-        v-e="['c:filter']"
-        :disabled="isLocked"
-        class="nc-filter-menu-btn nc-toolbar-btn !border-0 !h-7"
-        size="small"
-        type="secondary"
-      >
+      <NcButton v-e="['c:filter']" class="nc-filter-menu-btn nc-toolbar-btn !border-0 !h-7" size="small" type="secondary">
```
[the remark this answers is not in the document]
Comment (rameshmane7218) [reply packages/nc-gui/components/smartsheet/toolbar/ColumnFilterMenu.vue:76]: We removed from all toolbar button and disabling options from dropdown and giving unlock option in dropdown itself

**Review by dstala — APPROVED** [review]

**Comment by rameshmane7218** [comment]
Nothing to sync, changes alredy merge with pr #9864

**Pull request closed without merging**